iotfunctions/anomaly.py

```diff
@@ -0,0 +1,278 @@
+# *****************************************************************************
+# Â© Copyright IBM Corp. 2018.  All Rights Reserved.
+#
+# This program and the accompanying materials
+# are made available under the terms of the Apache V2.0
+# which accompanies this distribution, and is available at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# *****************************************************************************
+
+'''
+The Built In Functions module contains preinstalled functions
+'''
+
+import datetime as dt
```

Comment:
#import datetime as dt
#import time
#from collections import OrderedDict
#import skimage as ski
#import re
#import warnings
#import json

Good to remove unused imports